### ISSUE TITLE
using class="issue" will include githubusercontents URL

### ISSUE THREAD

**Issue by himorin** [speced/respec#3825]
## Important info

- spec: editing in PR / https://github.com/w3c/imsc-hrm/pull/11
- Respec version: ReSpec 26.16.0
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

we have a github issue with image attached and included in the initial comment, and we wanted to include it by using class="issue" feature of respec.
output from github API contains a img with source URL using githubusercontent, and that line causes an error on W3C publication.

it should be hard to fix that url and place contents locally during processing, so I'd request this to be documented...

**Comment by marcoscaceres** [comment]
>  that line causes an error on W3C publication.

@himorin, can you clarify a bit more what the problem is? 

It is because it's not on a w3c server?

**Comment by marcoscaceres** [comment]
Makes sense, thanks @plehegar ... I'll write something to check that images are either same origin (for development) or /TR/ and show a warning.
